test/support/helpers/DatabaseHelper.ts

```diff
@@ -0,0 +1,12 @@
+export default class DatabaseHelper {
+  createAdmin(): string {
+    const re = /[\n\r].*New administrator password is:s*([^\n\r]*)/;
+    let adminPassword = '';
+    cy.exec(
+      'sudo systemctl restart postgresql; sleep 2; yarn backend sequelize-cli db:drop; sleep 2; yarn backend sequelize-cli db:create; sleep 2; yarn backend sequelize-cli db:migrate; sleep 2; yarn backend sequelize-cli db:seed:all'
```

Comment:
Shouldn't need to restart postgresql here. I'm fine with the database being created and migrated before the tests run, and having this just seed the database and get the admin account over and over for every test.